### ISSUE TITLE
`vue/prop-name-casing` warns variable computed property

### ISSUE THREAD

**Issue by ota-meshi** [vuejs/eslint-plugin-vue#442]
**Tell us about your environment**

* **ESLint Version:**
4.18.1
* **eslint-plugin-vue Version:**
4.3.0
* **Node Version:**
8.0.0
**Please show your full configuration:**

```
    'vue/prop-name-casing': 'error',
```

**What did you do? Please include the actual source code causing the issue.**

```vue
        export default {
          props: {
            [greeting_text]: String
          }
        }

```

**What did you expect to happen?**

Since `[greeting_text]` is a variable computed property, it should not be a warn.

**What actually happened? Please include the actual, raw output from ESLint.**

`[greeting_text]` warned

```
Prop "greeting_text" is not in camelCase.
```

I noticed when I was PR #402.
https://github.com/vuejs/eslint-plugin-vue/pull/402#pullrequestreview-105602447
